src/Functions/randomPrintableASCII.cpp

```diff
@@ -120,7 +120,30 @@ class FunctionRandomPrintableASCII : public IFunction
 
 REGISTER_FUNCTION(RandomPrintableASCII)
 {
-    factory.registerFunction<FunctionRandomPrintableASCII>();
+    FunctionDocumentation::Description description = R"(
+Generates a string with a random set of [ASCII](https://en.wikipedia.org/wiki/ASCII#Printable_characters) characters.
```

Comment:
Generates a random [ASCII](https://en.wikipedia.org/wiki/ASCII#Printable_characters) string with the specified number of characters.